### ISSUE TITLE
Gotenberg 8

### ISSUE THREAD

**Issue by ssaunier** [MaikuMori/helm-charts#25]
First of all, thank you for maintaining this helm chart, very convinient to deploy Gotenberg to our favorite managed K8s cluster provider!

[Gotenberg 8.0.0 was released](https://github.com/gotenberg/gotenberg/releases/tag/v8.0.0) last week, any plan on making a new version of the helm chart?

Thank you very much.

**Comment by MaikuMori** [comment]
Absolutely, I was sailing across the Caribbean. I'm back on land now, this
is on the list to do in the next few days.

**Comment by MaikuMori** [comment]
@ssaunier It should be available from ArtifactHUB in about 12 minutes.

**Comment by ssaunier** [comment]
I managed to deploy it already as I saw it live on GitHub:

```bash
$ helm repo add maikumori https://maikumori.github.io/helm-charts
$ helm search repo maikumori/gotenberg
NAME               	CHART VERSION	APP VERSION	DESCRIPTION               
maikumori/gotenberg	1.0.0        	8.0.1      	A Helm chart for Gotenberg
```

My deployment went through, thanks for upgrading 🙏 !
